packages/core-js-compat/src/data.mjs

```diff
@@ -1760,6 +1760,8 @@ export const data = {
     node: '12.0', // '11.0',
     safari: '12.1',
   },
+  'web.structured-clone': {
```

Comment:
`deno: '1.14',`

Reply:
TODO: Add node.js 17.0.0 as well

Reply:
Deno supports `ArrayBuffer` transfers since v1.14.0, but does not support `Blob` and `File` serializations yet. You may need to patch it.
https://github.com/denoland/deno/issues/12067

Reply:
FF94 https://bugzilla.mozilla.org/show_bug.cgi?id=1722576